packster-core/src/operation/init_location.rs

```diff
@@ -42,8 +42,8 @@ impl InitLocationOperation<New> {
     }
 }
 
-pub fn abort_if_something_already_present<F: FileSystem, P: AsRef<Path>>(path_buff: P, filesystem: &F) -> Result<()> {
-    //TODO check if the location is not already present        
+pub fn abort_if_something_already_present<F: FileSystem, P: AsRef<Path>>(_path_buff: P, _filesystem: &F) -> Result<()> {
+    //TODO check if the location is not already present
     Ok(())
```

Comment:
You could maybe use some logging ? To just print something if there is something in the location.

Reply:
Something that is bothering me is that we could forgot to handle this case

Reply:
I usually solve all TODOs ( or tranform them into issues ) before releases, but ok